modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase

```diff
@@ -35,8 +35,13 @@ definition {
 	test ViewSystemObjects {
 		property portal.acceptance = "true";
 
-		// TODO LPS-137096 ViewSystemObjects pending implementation
+		var pAuth = Api.getPAuth();
+		var portalURL = PropsUtil.get("portal.url");
 
+		Navigator.openSpecificURL(url = "${portalURL}/group/guest/~/control_panel/manage?p_p_id=com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet&p_p_lifecycle=0&p_p_state=maximized&p_v_l_s_g_id=20125&p_p_auth=${pAuth}");
```

Comment:
Please, convert this to a macro inside the `ObjectAdmin.macro` since we will probably use this a lot in other tests.

Also, let's try to make it shorter and see if it works (removing the pAuth and part of the URL):

```
var portalURL = PropsUtil.get("portal.url");

Navigator.openSpecificURL(url = "${portalURL}/group/guest/~/control_panel/manage?p_p_id=com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet");
```